spec/parity/backup_spec.rb

```diff
@@ -15,6 +17,27 @@
       with(heroku_production_to_development_passthrough)
   end
 
+  it "correctly parses database.yml files that include ERB and environment variables" do
+    database_name = "parity_erb_development"
+    ClimateControl.modify DEVELOPMENT_DATABASE_NAME: database_name do
```

Comment:
what do you think about invoking this in an `around` block so that the tests aren't so deeply nested?